### DISCUSSION TITLE
Is username change immediate and available to be claimed by anyone?

### DISCUSSION THREAD

**Discussion by ishanavasthi** [community/community#63520] [Other Feature Feedback, Questions, & Ideas] [answered]
Question: Is username change immediate and available to be claimed by anyone?

### Body

If someone changes their username, is it instantly available for anyone else to claim (either by changing their username in account settings or creating a new account with that username).
I recently got admitted into a college and I am thinking to create a new fresh account with my college email address instead of adding the college email in my previous account for a fresh start. 
However, I would like to have my old username. Can I achieve this by just changing it in my previous account and making new account with that username. 
I am asking because in YouTube and some other sites if you change username neither can you change back to same username nor can anyone else claim it. It is basically lost.

**Comment by deyirao** [comment, marked as answer]
Hi avasthi-git!

I found this while researching about your problem:

"[...] After changing your username, your old username becomes available for anyone else to claim. Most references to your repositories under the old username automatically change to the new username. However, some links to your profile won't automatically redirect. [...]"

This is the information provided in the official GitHub site about changing your username.
https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-personal-account-settings/changing-your-github-username

If I understood this paragraph correctly, the username is not lost. Thus, there shouldn't be a problem.

I hope this helped ^^!

**Reply by ishanavasthi** [reply]
Thank you for the reply.

**Reply by ghost** [reply]
> Thank you for the reply.

Interested in doing the same thing, did it work?
